recipes-graphics/drivers/visionfive2-pvr-graphics_1.19.bb

```diff
@@ -40,7 +40,8 @@ do_install () {
     # let vulkan-loader from core layer provide libvulkan
     rm -rf ${D}${libdir}/libvulkan*.so* ${D}${libdir}/pkgconfig/vulkan.pc
     # provided via separate arch-independent firmware package
-    rm -rf ${D}/lib/firmware
+    # remove all firmware & systemd
+    rm -rf ${D}/lib/*
```

Comment:
can you add more info about the systemd files we are deleting ? Perhaps they can be used on systemd based distros and only deleted for non-systemd distro settings.